keyboards/alt67/alt67.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _ALT67_H_
```

Comment:
Could you replace the boiler plate with `#pragma once`?  We've been trying to switch over to that, as it's simpler/cleaner.